javascript/general_style_guide.md

```diff
@@ -41,7 +54,6 @@ We use the following config params:
 - `--single-quote`
 - `--trailing-comma es5`
```

Comment:
Do we still need to have these here? We already have the `prettier` configuration in our `package.json`

Reply:
Probably we don't need this. We can keep it here just for reference or get rid of this.

Reply:
@oponder since you wrote this, ¿Should we remove this or not?

Reply:
Yup, I'm ok with removing it.